bitnami/external-dns/templates/deployment.yaml

```diff
@@ -220,14 +220,19 @@ spec:
             - --rfc2136-port={{ .Values.rfc2136.port }}
             - --rfc2136-zone={{ required "rfc2136.zone must be supplied for provider 'rfc2136'" .Values.rfc2136.zone }}
             - --rfc2136-min-ttl={{ .Values.rfc2136.minTTL }}
-              {{- if .Values.rfc2136.tsigKeyname }}
-            - --rfc2136-tsig-secret-alg={{ .Values.rfc2136.tsigSecretAlg }}
-            - --rfc2136-tsig-keyname={{ .Values.rfc2136.tsigKeyname }}
-                {{- if .Values.rfc2136.tsigAxfr }}
+              {{- if .Values.rfc2136.tsigAxfr }}
```

Comment:
Previously we were adding this flag if `tsigKeyname` was also set and now we are adding it regardless of `tsigKeyname`. Why the change?

Reply:
As documented on the external-dns project itself https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/rfc2136.md `--rfc2136-tsig-axfr` is needed to delete records, even if our authentication method is Kerberos rather than a tsig key.